functions/src/download-readings.ts

```diff
@@ -103,16 +111,20 @@ async function generateReadingsCsv(
     // Contains readings for this sensor
     const readingsArray = new Array<string>(readingsList.length);
 
+    // TODO: Create separate files for different types of readings
     for (
       let readingIndex = 0;
       readingIndex < readingsList.length;
       readingIndex++
     ) {
-      const reading = SensorReading.fromFirestore(
-        readingsList[readingIndex].data()
-      );
+      const data = readingsList[readingIndex].data();
```

Comment:
Will these changes interupt @akampen 's work on the download? How does the system tell if it is a new or old reading/whether the call to `data.channelAPm25` is going to succeed or not?

Reply:
iirc, @akampen is currently testing downloads from the frontend. I created a bug to update this function though so that it will create two separate files with each reading type.

Reply:
Currently my code does not use this function, any changes are fine